### ISSUE TITLE
Strange volume mapping

### ISSUE THREAD

**Issue by duckietm** [madnuttah/unbound-docker#17]
Hello,

First of all thanks for your nice release of Unbound for docker 👍 
But why are the following volumes mapped ?

![image](https://user-images.githubusercontent.com/1917443/201915316-10c60f0d-c7ca-4f36-8462-7c5768e84ede.png)

my Config : 

```yml
version: "3"

services:
  unbound:
    container_name: unbound
    image: madnuttah/unbound:latest
    hostname: unbound
    networks:
     - dns-network
    ports:
      - 5335:5335/tcp
      - 5335:5335/udp
    environment:
      TZ: "Europe/Amsterdam"
    volumes:
      - ./unbound/unbound.conf:/usr/local/unbound/unbound.conf:rw #Your local path to Unbound
      - ./unbound/conf.d/:/usr/local/unbound/conf.d/:rw
      - ./unbound/log.d/:/usr/local/unbound/log.d/:rw
      - ./unbound/zones.d/:/usr/local/unbound/zones.d/:rw
      - ./unbound/certs.d/:/usr/local/unbound/certs.d/:rw
      - ./unbound/iana.d/:/usr/local/unbound/iana.d/:rw
    restart: unless-stopped

networks:
  dns-network:
    driver: bridge
```

How do i get rid of that [ and ] mapping ?

**Comment by madnuttah** [comment]
A pleasure! 😉 

You found an issue, this will be fixed in the next version. 

Thanks for your heads up!

**Comment by madnuttah** [comment]
The issue should be resolved with 1.17.0-4. Thanks again for reporting.
